### ISSUE TITLE
Add a link to the text downloads to the IIIF viewer

### ISSUE THREAD

**Issue by alexwlchan** [wellcomecollection/wellcomecollection.org#7592]
In our viewer for digitised content, we only offer a "Download PDF" option, e.g. https://wellcomecollection.org/works/jzdpnf8k

<img width="763" alt="Screenshot 2022-01-26 at 15 47 15" src="https://user-images.githubusercontent.com/301220/151196733-0df33405-28a1-48f9-b296-ffcdd11fd4de.png">

Previously we had a link to download all the OCR'd text for a digitised item, which was useful for screen reader users, especially in the library.

The API for that already exists (https://api.wellcomecollection.org/text/v1/b29411531) and is linked from the IIIF Presentation manifest (https://iiif.wellcomecollection.org/presentation/b29411531):

```
  "rendering": [
    ...
    {
      "id": "https://api.wellcomecollection.org/text/v1/b29411531",
      "type": "Text",
      "label": {
        "en": [
          "View raw text"
        ]
      },
      "format": "text/plain"
    }
  ],
```

We should add a link to the download button for this text file.

**Comment by gestchild** [comment]
#4084 and #4076 are relevant to this.

I think the rationale for not including the text was that it was rarely used (and therefore created noise around the other options?) see #4076

Not saying we shouldn't do this, I think the accessibility use case is important, just adding for information.

**Comment by jennpb** [comment]
Alternative use case for OCR is researchers who want to analyse the full text. Will close this now due to lack of evidenced user need.
